admin/code/LeftAndMain.php

```diff
@@ -536,11 +536,10 @@ protected function init() {
 			window.ss.config = " . $this->getCombinedClientConfig() . ";
 		");
 
-		Requirements::javascript(ltrim(FRAMEWORK_ADMIN_DIR . '/client/dist/js/bundle-lib.js', '/'));
+		Requirements::javascript(FRAMEWORK_ADMIN_DIR . '/client/dist/js/vendor.js');
```

Comment:
It's not strictly needed for FRAMEWORK_ADMIN_DIR. I put this in for the glorious future where admin is also a separate module and we're running tests from its root with framework as a subfolder. ;-)

I wouldn't block merge for this because I didn't consistently apply it to all instances of FRAMEWORK_ADMIN_DIR, and it's a temporary code pattern anyway.